core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -216,6 +216,7 @@ const getTopicEditorUrl = async function(browser, page) {
   }
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can you avoid adding this in the first place by replacing the unused var with _ or prefixing it with "unused" or something like that?

Reply:
Could you give an example I tried to prefix it with unused but it still fails the lint check

Reply:
Sorry, scratch that. I looked at .eslintrc and it looks like we don't have such a construct yet.

However, I took a look at https://github.com/oppia/oppia/pull/10441/files and it seems we don't have checks for such variables. So I'm not sure if you're fixing the correct thing. Did you run the lint checks locally to see if they work? (Look at scripts/linters/pre_commit_linter.py.)

Reply:
A few questions:

- If getStoryEditorUrl is unused, why have it at all? (why not just delete the function entirely?)
- Is the issue that getStoryEditorUrl is unused, or that "browser" is unused? If the latter, perhaps just don't pass "browser" in?

Usually, the solution is **not** to add a new eslint-disable.